src/capture/mod.rs

```diff
@@ -13,6 +14,7 @@ pub enum CaptureArgs {
     /// Capture open window information from a (linux) X11 server
     X11(X11CaptureArgs),
     Windows(WindowsCaptureArgs),
+    OSX(OSXCaptureArgs),
```

Comment:
why are you calling it OSX? from what i understand, macos is the official name since a few years ago